index.html

```diff
@@ -25,27 +29,255 @@
+                        </div>
+                        <div class="primary-text_text">
+                            <p>
+                                A daily selection of privately
+                                personalized reads; no accounts or
+                                sign-ups required.
+                            </p>
+                        </div>
+                        <div class="tags">
+                            <ul>
+                                <li class="tag">html</li>
+                                <li class="tag">css</li>
+                                <li class="tag">javaScript</li>
+                            </ul>
+                        </div>
+                        <div class="action">
+                            <button type="button">See Project</button>
+                        </div>
+                    </div>
+                </div>
+
+            </section>
+
+            <div class="about">
+                <div class="top">
+                    <h2 class="text-primary">About <br> Myself</h2>
+                    <p class="text-secondary">
+                        Hello I’m a software developer! I can help you build a product,
+                        feature or website Look through some of my work and experience! If you like what you see and
+                        have a project you need coded,
+                        don’t hestiate to contact me.
+                    </p>
+                    <div class="social-media">
+                        <div class="supporting-text">
+                            <p>LET'S CONNECT</p>
+                        </div>
+                        <div class="frame1">
+                            <ul class="social-icon-list">
+                                <li><a href="https://twitter.com/kusilaw"><img src="./images/twitter.svg"
+                                            alt="twitter"></a></li>
+                                <li><a href="#"><img src="./images/Linkedin-icon.svg" alt="link"></a></li>
+                                <li><a href="#"><img src="./images/like.svg" alt="like"></a></li>
+                                <li><a href="#"><img src="./images/man.svg" alt="man"></a></li>
+                                <li><a href="https://github.com/kusiLaw"><img src="./images/github.svg"
+                                            alt="github"></a></li>
+                            </ul>
+                        </div>
+                    </div>
+                    <button type="button" class="secondary">Get my resume</button>
+                </div>
+
+                <ul class="skill-list">
+                    <li class="frame12">
+                        <div class="frame7">
+                            <!-- <img class="btn-separator" src="./images/Separator-Bottom.png" alt=""> -->
+                            <span>Languages</span>
+                            <button type="button" class="normal-btn"> &#8964; </button>
```

Comment:
The icons' size doesn't match the template design. I recommend you download the icons directly from the Figma template and apply them to this section.

Reply:
thank you very much, I have been trying to export but anytime I export the icon from figma it is empty.
will it be allowed, if I increase its font size to 24px to have the same sizes as the one on figma?.
 thank you